docker/scripts/context-path-adapter.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
```

Comment:
Could add `u` as well, to error on undefined env vars.

Reply:
Sure, but there is only one variable, which is set by default into the Dockerfile.